composeApp/src/androidMain/kotlin/com/jetbrains/spacetutorial/AppViewModel.kt

```diff
@@ -0,0 +1,35 @@
+package com.jetbrains.spacetutorial
+
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.jetbrains.spacetutorial.entity.RocketLaunch
+import kotlinx.coroutines.launch
+
+class AppViewModel(private val sdk: SpaceXSDK) : ViewModel() {
+
+    private val _state = mutableStateOf(RocketLaunchScreenState())
+    val state: State<RocketLaunchScreenState> = _state
```

Comment:
Nit on naming: I would expect that `RocketLaunchScreenState` is in a `RocketLaunchViewModel`, or something similar to that